Adding Commands to LG Driver

eduardoveiga opened the pull request:
key_home
    key_up
    key_left
    key_enter
    key_right
    key_down
    key_esc
    key_a
    key_z
    key_r
    volume_up
    volume_down

Signed-off-by: Eduardo Kluwe Veiga <eduardoveiga@protonmail.ch>

eduardoveiga committed:
Adding Commands to LG Driver
    key_home
    key_up
    key_left
    key_enter
    key_right
    key_down
    key_esc
    key_a
    key_z
    key_r
    volume_up
    volume_down

Signed-off-by: Eduardo Kluwe Veiga <eduardoveiga@protonmail.ch>

eduardoveiga commented:
## observations

1. In the reference manual, there's no volume up and down commands, just volume set, but in  the IR codes there key codes to volume  up and volume down keys. I don't know if this is the same.

2. In the code, I changed the `strconv.Atoi(var)` to `strconv.ParseInt(var, 16, 0)`  because the IR and DATA values, in the command string, are in hexadecimal format

otavio reviewed (changes requested):
The PR looks good; the only remark is that this must be split.

1) patch fixing the parsing (regex and hex)
2) the new commands

Please fix this and force-push it here so we review it one more time.

eduardoveiga committed:
Fixing regex and Parseint functions from decimal to Hex

 changed the strconv.Atoi(var) to strconv.ParseInt(var, 16, 0) because the IR and DATA values, in the command string, are in hexadecimal format

eduardoveiga commented:
reopening the pull request that I close by accident
